plugin.js

```diff
@@ -105,8 +35,6 @@ function cmifyPlugin (b, opts) {
   const cmStream = createCmStream()
   b.require(cmStream, { expose: cmStream.id })
 
-  b.transform(cmifyTransform)
```

Comment:
Stops the plugin registering the transform implicitly, requiring the user to explicitly register it with a `-t cmify/transform` flag to browserify.